apps/mark-scan/backend/src/app.ts

```diff
@@ -111,8 +111,7 @@ export function buildApi(
     },
 
     unconfigureMachine() {
-      workspace.store.setElectionAndJurisdiction(undefined);
-      workspace.store.deleteSystemSettings();
+      workspace.store.reset();
     },
```

Comment:
> TODO for later: noticed the "unconfigure" endpoints are unguarded WRT user type, so might be worth adding an "is EM or SysAdmin" check, at least for consistency with the "configure" endpoints.

Makes sense! I've been wanting to add some kind of auth decorators, so that we can easily add access checks across app endpoints: https://github.com/votingworks/vxsuite/issues/3679

Reply:
Ahh, nice - having some easy/consistent way of annotating endpoints would be great!